content/guides/guides/debugging.md

```diff
@@ -25,16 +25,28 @@ browser makes available to you, like `document`, `window`, and `debugger`.
 Based on those statements, you might be tempted to throw a `debugger` into your
 test, like so:
 
+:::visit-mount-test-example
```

Comment:
Please ensure the content after this example mentions `cy.mount()` where relevant.

Reply:
This is covered in https://github.com/cypress-io/cypress-documentation/pull/4207/files